backend/src/yeomanui.ts

```diff
@@ -154,7 +154,16 @@ export class YeomanUI {
           console.log(`image contents: ${image}`);
         }
       }
-
+      
+      const genInstall = _.get(gen, "install");
+      if (genInstall) {
+        var that = this;
+        gen.__proto__.install = function() {
+          that.doGeneratorInstall();
```

Comment:
change to:

const originalGenInstall = _.get(gen, "install");
      if (originalGenInstall) {
        gen.install = () => {
        this.doGeneratorInstall();
        originalGenInstall.call(gen);
        };
      }

Reply:
Done